sonic-utilities-tests/intfstat_test.py

```diff
@@ -103,17 +105,17 @@ def test_clear(self):
         assert result.exit_code == 0
         result = runner.invoke(show.cli.commands["interfaces"].commands["counters"].commands["rif"], [])
         print(result.stdout)
-        expected = ["Ethernet20        0  0.00 B/s    0.00/s         0        0  0.00 B/s    0.00/s         0",
-                    "PortChannel0001        0  0.00 B/s    0.00/s         0        0  0.00 B/s    0.00/s         0",
-                    "PortChannel0002        0  0.00 B/s    0.00/s         0        0  0.00 B/s    0.00/s         0",
-                    "PortChannel0003        0  0.00 B/s    0.00/s         0        0  0.00 B/s    0.00/s         0",
-                    "PortChannel0004        0  0.00 B/s    0.00/s         0        0  0.00 B/s    0.00/s         0",
-                    "Vlan1000        0  0.00 B/s    0.00/s         0        0  0.00 B/s    0.00/s         0"]
+        expected = ["Ethernet20   0  0.00 B/s   0.00/s     0.00%         0         0  0.00 B/s    0.00/s     0.00%         0",
```

Comment:
what will be command to check RIF  Counters ? 
Has this been verified if RIF is both Over Port-Channel and VLAN ?

Reply:
The command to check RIF counters is `show interface counters rif` .